preDeploy.sh

```diff
@@ -0,0 +1,36 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -xe
+echo "pre deploy code goes here"
+
+echo "IDS_PROJECT_NAME=${IDS_PROJECT_NAME}"
+
+#templateBranch="master"
```

Comment:
Can we delete some commented out code here?
This `pr` is massive, and it would go a long way in making it more consumable.

Reply:
I opted to leave it in for now. Since the toolchain work is not 100% complete, a lot of this will need to be uncommented and modified to work with our project once we set up our helm overrides for our different deployment environments (ie CSP) and I didn't want to have to go hunt it down.

Reply:
okay, I think that we should add a `todo` here with a note to give some context to the people not as familiar toolchains.

Reply:
fair point, I'll add some verbage